tensorflow/python/compiler/tensorrt/trt_convert_test.py

```diff
@@ -837,6 +837,18 @@ def testTrtGraphConverter_StaticOp(self):
         # to fall back to TF function.
         self._TestRun(sess, 2)
 
+  @test_util.run_v2_only
+  def testBackwardCompatibility(self):
+    """Load and execute a model that was saved in TF2.0."""
+    if not is_tensorrt_enabled():
+      return
+
+    model_dir = test.test_src_dir_path(
+        'python/compiler/tensorrt/test/testdata/tftrt_2.0_saved_model')
```

Comment:
This PR also checks in the variables of the test model, is that required even after the conversion?

Reply:
I just added check for correctness which would need the variables.